### ISSUE TITLE
`7.4.4` stops a `WKNavigationDelegate` function from being called by a web view

### ISSUE THREAD

**Issue by adamaszhu** [newrelic/newrelic-ios-agent-spm#31]
## Description

`func webView(_: WKWebView, decidePolicyFor navigationAction: WKNavigationAction, decisionHandler: @escaping (WKNavigationActionPolicy) -> Void)` stops getting called with `7.4.4`, which was working with `7.4.3`.

## Steps to Reproduce

- Check out a dev branch that has a working `func webView(_: WKWebView, decidePolicyFor navigationAction: WKNavigationAction, decisionHandler: @escaping (WKNavigationActionPolicy) -> Void)` function for a web view
- Upgrade New Relic only from `7.4.3` to `7.4.4`
- This function won't get called. And instead, any links in a web view will just be opened in the web view

## Expected Behavior

The method should be called and we can perform our customized actions for links

## Relevant Logs / Console output

```
NewRelic(7.4.4,0x600002665d00):	NRMAUncaughtExceptionHandler.m:63	-[NRMAUncaughtExceptionHandler start]_block_invoke
New Relic Crash Reporting is DISABLED because it has detected the debugger is enabled.
```

## Your Environment

Real device: iPhone XS + iOS 16.3.1
Simulator: iPhone 7 + iOS 13.7

**Comment by MeteC** [comment]
Lost a LOT of time to this bug, breaking as it did a seemingly completely unrelated dependency (and resulting in many  very upset customers.) Literally stumbled on it as the root cause of our issues by pure chance, only to then discover this bug has been raised 2 weeks ago. Hate to be that guy, but - NewRelic is supposed to detect issues, not cause them!

Hope it's being taken seriously and attended to swiftly!

**Comment by smalsam-newr** [comment]
@MeteC @adamaszhu We have been made aware of this. We do have a fix in the works, which should be coming with our next release coming in the next week or two.

**Comment by mbruin-NR** [comment]
@MeteC @adamaszhu  Agent version 7.4.5 was released, can you confirm that this bug is fixed?

**Comment by garethlloyd1994** [comment]
@mbruin-NR - can confirm this bug is fixed for us in 7.4.5 :)

**Comment by MeteC** [comment]
Also confirming this seems fine on our end, thanks team!
